apps/sap-commerce-cloud/app-actions/src/actions/fetchProductList.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import {
+  makeMockAppActionCallContext,
+  makeMockAppInstallation,
+  makeMockFetchResponse,
+  mockSapProductList,
+} from '../../test/mocks';
+import { AppInstallationProps } from 'contentful-management';
+import { AppActionCallContext } from '@contentful/node-apps-toolkit';
+import { handler } from './fetchProductList';
+import { AppActionCallResponseSuccess } from '../types';
+
+describe('fetchProductList.handler', () => {
+  let cmaRequestStub: sinon.SinonStub;
+  let context: AppActionCallContext;
+  const cmaClientMockResponses: [AppInstallationProps] = [makeMockAppInstallation()];
+  let stubbedFetch: sinon.SinonStub;
+
+  beforeEach(() => {
+    const mockFetchResponse = makeMockFetchResponse(mockSapProductList);
+    stubbedFetch = sinon.stub(global, 'fetch');
```

Comment:
Per David's feedback in the `baseSites` PR, maybe add something to restore fetch? https://github.com/contentful/apps/pull/8620/files#r1723521458